lib/DeonRestClient.ts

```diff
@@ -7,7 +7,14 @@ import {
   NotFoundError,
   BadRequestError,
 } from './DeonApi';
-import { InstantiationInput, ExpressionInput, DeclarationInput, Event, Tag } from './DeonData';
+import {
+  InstantiationInput,
+  CheckExpressionInput,
+  EvaluateExpressionInput,
+  DeclarationInput,
+  Event,
+  Tag
```

Comment:
You need a trailing comma here to make the linter happy.

Reply:
Done.